angular/openmrs-translate/openmrs-translate.module.js

```diff
@@ -25,8 +26,11 @@ function translateConfig(openmrsTranslateProvider) {
 openmrsTranslateProvider.$inject = ['$translateProvider'];
 function openmrsTranslateProvider($translateProvider) {
     function init() {
-        $translateProvider.fallbackLanguage('en');
-        $translateProvider.preferredLanguage('en');
+        $translateProvider.fallbackLanguage('en')
+            .preferredLanguage('en')
+            .useUrlLoader('/' + OPENMRS_CONTEXT_PATH + '/module/uicommons/messages/messages.json')
```

Comment:
https://demo.openmrs.org/openmrs/module/uicommons/messages/messages.json doesn't seem to be working... even after including ?lang=en. The error smells like an issue with openmrs-core 2.x+ Are you running some lower version? Ideally messages would be exposed by the webservices.rest module and we would use openmrsRest angular service to get the context path instead of the globally defined constant...